src/SFA.DAS.Commitments.Database/StoredProcedures/GetApprenticeshipsByULN.sql

```diff
@@ -0,0 +1,16 @@
+﻿CREATE PROCEDURE [dbo].[GetApprenticeshipsByULN]
+(
+  @ULN NVARCHAR(50)
+)
+AS
+
+SELECT 
+	s.*
+	FROM ApprenticeshipSummary s
+	WHERE s.ULN = @ULN
+	ORDER BY s.FirstName asc, s.LastName asc;
+
+SELECT 
```

Comment:
Again, to reduce workload on sql server, we could perform the count in the api/website as required (or at least use @@ROWCOUNT).

Reply:
Will make some changes